uk.ac.kcl.inf.modelling.xdsml.gemoc_henshin/src/uk/ac/kcl/inf/modelling/xdsml/gemoc_henshin/engine/ui/launcher/tabs/LaunchConfigurationBackendsTab.xtend

```diff
@@ -7,7 +7,11 @@ import org.eclipse.gemoc.xdsmlframework.api.extensions.engine_addon.EngineAddonS
 import org.eclipse.gemoc.xdsmlframework.api.extensions.engine_addon_group.EngineAddonGroupSpecificationExtension
 import org.eclipse.gemoc.xdsmlframework.api.extensions.engine_addon_group.EngineAddonGroupSpecificationExtensionPoint
 
-// Copied from original GEMOC code to work around package export issue

```

Comment:
This should stay and we should add an issue to their repo to ask for this to be made available.